CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@ contributors and maintainers.
 
 [Slack](https://kuma-mesh.slack.com) is the main chat channel used by the
 community and the maintainers of this project. If you do not have an
-existing account, please follow this [link](https://join.slack.com/t/kuma-mesh/shared_invite/zt-1rcll3y6t-DkV_CAItZUoy0IvCwQ~jlQ) to sign
+existing account, please follow this [link](https://join.slack.com/t/kuma-mesh/shared_invite/zt-1tu98qcw4-pJNW9lRdBqx1I9kF9rvImA) to sign
```

Comment:
No idea why the old one expired (I did not do anything with my account, do they expire base on time?). @lahabana - should we bring back the invite bot somehow?

Reply:
Also this link was present in multiple places so need to update all of them 😢

Reply:
Ok, now that I re-generated the link it said that it expires in 30 days. I'm 99% sure that was not the case last time I generated it.

![image](https://github.com/kumahq/kuma/assets/2753650/439510dc-7dfe-49d6-baef-8d91e25316a8)

We probably don't want to repeat this every month so probably need to invest in something like https://communityinviter.com/ ?